### PR TITLE
Unshit indents and docs

### PR DESCRIPTION
**Pull request by aurieh**
Remind me to rewrite tests while I'm at it.

### CODE REVIEW

**Comment by deansheather** [comment]
Rewrite tests while you're at it.

**Comment by aurieh** [comment]
👌

File: src/owo.js [hunk 1]
```diff
@@ -1,34 +1,37 @@
...
 const Endpoints = require('./Endpoints');
 const request = require('superagent');
 const OwOError = require('./owoError');
 
-/*
- * Creates a new class
- * @prop {key} key your whats-th.is API Key.
- */
+/**
+  * OwO Client
```
Comment (sr229) [inline src/owo.js:13]: This is JSDoc, specify what OwO Client does (does it create?)

**Review by sr229 — DISMISSED** [review]

File: src/owo.js (hunk 1 above)
Comment (aurieh) [reply src/owo.js:13]: But this is JSDoc for the class, you document what it creates on constructor below.

Comment (sr229) [reply src/owo.js:13]: that's the point, sparky

Comment (aurieh) [reply src/owo.js:13]: I still don't get what changes you want from this. I followed the standard, documenting constructor on the constructor, and class description as the top doc comment.

Comment (sr229) [reply src/owo.js:13]: Explain it **thoroughly** like ``Creates an OwO Client`` or smth

Comment (aurieh) [reply src/owo.js:13]: Line 12, `Construct a new Client`

Comment (sr229) [reply src/owo.js:13]: o

**Pull request merged**